client/tasks/options/watch.js

```diff
@@ -19,7 +19,7 @@ module.exports = {
     },
     assets: {
         options: {livereload: true},
-        tasks: ['ngtemplates'],
+        tasks: [''],
```

Comment:
With this change, does it build template.js in local when there is a change in html?